windows/src/engine/keyman32/k32_globals.cpp

```diff
@@ -379,6 +384,9 @@ void Globals::SetBaseKeyboardFlags(char *baseKeyboard, BOOL simulateAltGr, BOOL
   be changed until Keyman is restarted.
 */
 BOOL Globals::InitSettings() {
+  /* Check for common core vs windows core */
+  f_CoreIntegration = Reg_GetDebugFlag(REGDW_CoreIntegration, TRUE);
```

Comment:
Technically, we still say `REGSZ_foo` because the `SZ` refers to the type of the `#define` ... though what you did would have been better, let's keep with tradition here for now :)

Reply:
haha ok the tradition had me confused for a while as the ReadInteger.